platform-ajax/src/main/resources/com/softicar/platform/ajax/engine/TypeScriptGenerated.js

```diff
@@ -746,7 +739,7 @@ class KeyboardEventHandler {
         this.listenToKeyDown = false;
         this.listenToKeyUp = false;
         this.listenToKeys = new Set();
-        this.lastKeyDown = 0;
+        this.lastKeyDown = "0";
```

Comment:
i think this should be rather `""` since only that does not match any potential key